tests/localosmosis/scripts/setup.sh

```diff
@@ -88,7 +88,8 @@ edit_genesis () {
 add_genesis_accounts () {
 
     osmosisd add-genesis-account osmo12smx2wdlyttvyzvzg54y2vnqwq2qjateuf7thj 100000000000uosmo,100000000000uion,100000000000stake,100000000000uusdc,100000000000uweth --home $OSMOSIS_HOME
-    osmosisd add-genesis-account osmo1cyyzpxplxdzkeea7kwsydadg87357qnahakaks 100000000000uosmo,100000000000uion,100000000000stake,100000000000uusdc,100000000000uweth --home $OSMOSIS_HOME
+    # note such large amounts are set for e2e tests on FE 
+    osmosisd add-genesis-account osmo1cyyzpxplxdzkeea7kwsydadg87357qnahakaks 9999999999999999999999999999999999999999999999999uosmo,9999999999999999999999999999999999999999999999999uion,100000000000stake,100000000000uusdc,100000000000uweth --home $OSMOSIS_HOME
```

Comment:
This doesn't effect max supply at all?

Reply:
I'm unsure of the question - could you elaborate?

I think `add-genesis-account` command should update all state that is necessary.

Reply:
yes, AFIAK add-genesis-account is safe (handles max suppy at time of gentx)